carbon-design-system/carbon-addons-iot-react
# chore(repo): initial visual regression images commit

Closes #2867

**Summary**

- Just adding an initial commit of regression images to see if they pass in CI.

**Change List (commits, features, bugs, etc)**

- Adds Dockerfile and docker-compose.yml to root
- Adds `test:e2e:docker:build` script to build the container
- adds `test:e2e:docker:images` script to generate the visual regression images
- adds `test:e2e:docker:clean` to run `docker system prune` to clean up when the docker image is full
- adds simple documentation to explain the regression test process
- updates github actions to use the same container and changes the existing steps as necessary to account for this new container
- makes sure all width/height are set to 1680/900 respectively.

**Acceptance Test (how to verify the PR)**

- install docker on your machine
- create a new docker machine
- start the new machine
- run `yarn test:e2e:docker:build` from the `packages/react` directory to build the new image
- run `yarn test:e2e:images` to run e2e tests that generate the images
- tests should pass for Button/CardCodeEditor

**Regression Test (how to make sure this PR doesn't break old functionality)**

- ensure all github actions still run correctly
- no adverse effects.



**Things to look for during review**
